google/cloud/security/common/gcp_api/compute.py

```diff
@@ -656,6 +678,28 @@ def get_forwarding_rules(self, project_id, region=None):
                                                        'forwardingRules')
         return results
 
+    def get_global_operations(self, project_id, operation_id):
```

Comment:
Nit: Change name to get_global_operation since this only gets a single operation.